ale_linters/tex/textidote.vim

```diff
@@ -0,0 +1,47 @@
+" Author: Jordi Altayo <jordiag@kth.se>, Juan Pablo Stumpf
+" <juanolon@gmail.com>
+" Description: support for textidote grammar and syntax checker
+
+call ale#Set('tex_textidote_executable', 'textidote')
+call ale#Set('tex_textidote_options', '--no-color --output singleline')
+" TODO get language from spell spelllang
+call ale#Set('tex_textidote_check_lang', '')
+
+
+function! ale_linters#tex#textidote#GetExecutable(buffer) abort
+    let l:exe = ale#Var(a:buffer, 'tex_textidote_executable')
+    let l:exe .= ' ' . ale#Var(a:buffer, 'tex_textidote_options')
+
+    let l:check_lang = ale#Var(a:buffer, 'tex_textidote_check_lang')
```

Comment:
Small attempt to get language from spelllang configuration, may be worth considering:

```
" Function tries to determine the language to use with textidote from the
" spelllang configuration.
function! ale_linters#tex#textidote#GetSpellLang(buffer) abort

  let l:check_lang = ale#Var(a:buffer, 'tex_textidote_check_lang')

  if !empty(l:check_lang)
    return l:check_lang
  endif

  let l:supported_languages = ['en', 'de', 'es', 'fr', 'nl', 'pt']
  let l:spell_languages = split(&spelllang, ',')

  for spell_lang in l:spell_languages
    for supported_lang in l:supported_languages
      if(spell_lang == supported_lang)
        return spell_lang
      endif
    endfor
  endfor

  return ''
endfunction
```